frontera/core/manager.py

```diff
@@ -325,13 +336,13 @@ def get_next_requests(self, max_next_requests=0, **kwargs):
                                             (max_next_requests, self.n_requests, self.max_requests or '-')))
 
         # get next requests
-        next_requests = self.backend.get_next_requests(max_next_requests, **kwargs)
+        next_requests = self.backend.get_next_requests(max_next_requests, **kwargs) or []
```

Comment:
May be it makes more sense to force backends to return only lists? This code allows to return 0, None, False, etc.